documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -47,7 +47,7 @@ Default: `115200`
 
 If `1` then `sudo halt` will run in a lower power mode until either GPIO3 or GLOBAL_EN are shorted to ground.
 
-This setting is not relevant on Raspberry Pi 5 because the xref:raspberry-pi.adoc#power-button[dedicated power button] may always be used to wakeup from `HALT` or `STANDBY`.
+This setting is not relevant on Flagship models since 5, Compute Modules since 5, and Keyboard models since 500 because the xref:raspberry-pi.adoc#power-button[dedicated power button] may always be used to wake from `HALT` or `STANDBY`.
```

Comment:
```suggestion
This setting is not relevant on Flagship models since Raspberry Pi 5, Compute Modules since 5, and Keyboard models since 500 because the xref:raspberry-pi.adoc#power-button[dedicated power button] may always be used to wake from `HALT` or `STANDBY`.
```